docs/news.d/optional_annotation.rst

```diff
@@ -1,6 +1,6 @@
 .. news-prs: 4364
 
 .. news-start-section: Additions
-- Added parsing of the @optional annotation to opendds_idl. @optional maps to either std::optional or a similar custom implementation in the generated code.
+- Added support for the :ref:`xtypes--optional` IDL annotation for the C++11-to-IDL mapping. ``@optional`` maps to either ``std::optional`` or a similar custom implementation in the generated code.
```

Comment:
```suggestion
- Added support for the :ref:`xtypes--optional` IDL annotation for the IDL-to-C++11 mapping. ``@optional`` maps to either ``std::optional`` or a similar custom implementation in the generated code.
```

Also, do the + characters in C++11 need special handling / escaping for RST?

Reply:
> C++11-to-IDL mapping.

I mixed those up...

> do the + characters in C++11 need special handling / escaping for RST?

No, [it looks fine](https://opendds--4364.org.readthedocs.build/en/4364/news.html#additions). If I'm reading [this](https://docutils.sourceforge.io/docs/user/rst/quickref.html#inline-markup) correctly, I think only `*`, `` ` ``, `|`, and `_` need to be escaped. I imagine you'd also need to escape `\` sometimes.